microservices/Keli-Katheya-User-Registration/src/main/java/com/kelikatheya/microservices/userRegistration/requestModels/UserRequest.java

```diff
@@ -0,0 +1,43 @@
+package com.kelikatheya.microservices.userRegistration.requestModels;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class UserRequest {
+    @NotNull(message = "Name can't be empty")
+    @Size(min = 5 , message = "Minimum of 5 Characters required")
+    private String name;
+
+    @NotNull(message = "email can't be empty")
+    @Email
+    private String email;
+
+    @NotNull(message = "password can't be empty")
+    @Size(min = 6 , max = 15 , message = "Password size should be between 5 and 15")
```

Comment:
Why max limit for passwords?

Reply:
removed max limit, requires discussion to keep max length for password